tests/cpp/test_aten.cc

```diff
@@ -144,7 +144,7 @@ TEST(ArrayTest, TestHStack) {
 template <typename IDX>
 void _TestIndexSelect() {
   IdArray a = aten::Range(0, 100, sizeof(IDX)*8, CTX);
-  ASSERT_EQ(aten::IndexSelect(a, 50), 50);
+  ASSERT_EQ(aten::IndexSelect<int>(a, 50), 50);
```

Comment:
How about the other array operators you added? Have you tested them?

Reply:
They are tested in python function `test_pack_traces()`